src/View/mainpage/NewStructureTab.py

```diff
@@ -0,0 +1,279 @@
+import csv
+from pathlib import Path
+from random import randint, seed
+
+from PyQt5 import QtWidgets, QtGui, QtCore
+from pandas import np
+
+from src.Controller.ROIOptionsController import RoiDeleteOptions, ROIDelOption, ROIDrawOption
+from src.Model import ImageLoading
+from src.Model.PatientDictContainer import PatientDictContainer
+from src.Model.ROI import ordered_list_rois
+from src.View.mainpage.StructureWidget import StructureWidget
+
+
+class NewStructureTab(QtWidgets.QWidget):
```

Comment:
let's hope there isn't a NewNewStructureTab in the future.
Maybe the name could contain something that indicates how this is different from the "old" StructureTab ?